main_test.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/env"
+	"github.com/bitrise-io/go-xcode/utility"
+	"github.com/bitrise-io/go-xcode/xcarchive"
+)
+
+func TestConfig_generateExportOptions_plist(t *testing.T) {
+	// Given
+	envRepository := env.NewRepository()
+	commandFactory := command.NewFactory(envRepository)
+	xcodebuildVersion, _ := utility.GetXcodeVersion(commandFactory)
+	archive, _ := xcarchive.NewIosArchive("configs.ArchivePath")
+
+	// When
+	result, _ := generateExportOptionsPlist("app", "development", "my team id", false, false, xcodebuildVersion.MajorVersion, archive, false)
+
+	// Then
+	if len(result) == 0 {
```

Comment:
This could be easier expressed with functions from the `assert` package, for example:

```go
assert.Equal(t, 0, len(result))
```

You can also replace other assertions below

Reply:
nice one.. updated